wee_slack.py

```diff
@@ -583,6 +584,10 @@ def buffer_prnt(self, user='unknown_user', message='no message', time=0):
                     message = user.name_regex.sub(
                         r'\1\2{}\3'.format(user.formatted_name() + w.color(chat_color)),
                         message)
+
+            if replace_emoji:
+                message = emoji.emojize(message, use_aliases=True)
```

Comment:
You probably want to call this when you get a reply to a message you sent too, that currently doesn't go through buffer_prnt.